sys.py/UI/untitled_icon.py

```diff
@@ -11,34 +11,36 @@
 import base64
 from beeprint import pp
 
-from util_funcs  import midRect,SkinMap
-from fonts       import fonts
+from util_funcs import midRect, SkinMap
+from fonts import fonts
 
-BlankPng = SkinMap("gameshell/blank.png")  ## 80x80
+from skin_manager import SkinManager
+
+BlankPng = SkinMap("gameshell/blank.png")  # 80x80
 ## use blank circle as bg, Two alpha As Icon Label
 #Upper and Lower
+
+
 class UntitledIcon(object):
     _PosX = 0
     _PosY = 0
     _Width = 80
     _Height = 80
-    
-    _Words = ["G","s"]
-    _FontObj= fonts["varela40"]
-    
-    _BG     = None  ## initial surface 
 
-    _Color  = pygame.Color(83,83,83)
+    _Words = ["G", "s"]
+    _FontObj = fonts["varela40"]
+
+    _BG = None  # initial surface
+
+    _Color = SkinManager().GiveColor('Text')
```

Comment:
I'm going to re-review this file. Git seems to think it changed more than it did.